OpenRA.Mods.Common/Traits/Sound/DeathSounds.cs

```diff
@@ -41,7 +40,7 @@ void INotifyKilled.Killed(Actor self, AttackInfo e)
 			if (IsTraitDisabled)
 				return;
 
-			if (Info.DeathTypes.Count == 0 || e.Damage.DamageTypes.Overlaps(Info.DeathTypes))
+			if (!Info.DeathTypes.IsEmpty || e.Damage.DamageTypes.Overlaps(Info.DeathTypes))
```

Comment:
This should've been `Info.DeathTypes.IsEmpty` (without '!'), if I'm reading this correctly.